webapp/lib/controller.dart

```diff
@@ -90,6 +99,23 @@ void initUI() {
   );
 }
 
+void checkNeedsReplyMetricsPipelineDataFlow() {
+  var sortedNeedsReplyDataList = List.from(needsReplyDataList);
+  sortedNeedsReplyDataList.sort((d1, d2) => d1.datetime.compareTo(d2.datetime));
+  var lastNeedsReplyEntry = sortedNeedsReplyDataList.where((entry)=>
+      entry.project == view.contentView.projectSelectorView.selectedProject).last;
+
+  var now = new DateTime.now();
+  var lastUpdateTimeDiff =  now.difference(lastNeedsReplyEntry.datetime).inHours;
+
+  if (lastUpdateTimeDiff > 1) {
+    view.contentView.conversationCharts.forEach((chart) => chart.classes.add('stale'));
+  } else {
+    view.contentView.conversationCharts.forEach((chart) => chart.classes.remove('stale'));
```

Comment:
I prefer leaving details of how the stale display is implemented to the view, and here have just something like:
```
if (lastUpdateTimeDiffInHours > 1) {
  view.contentView.stale = true;
} else {
  view.contentView.stale = false;
}
```

Reply:
ok...good separation of concern

Reply:
Implemented the stale logic in the view